src/main/java/com/osp/dragon/service/ArticlesReader.java

```diff
@@ -0,0 +1,35 @@
+package com.osp.dragon.service;
+
+import com.osp.dragon.model.Article;
+import org.springframework.stereotype.Service;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class ArticlesReader {
+
+    private static final String FILE_NAME = "C:/Users/Raczkowski/IdeaProjects/dragon2/src/main/resources/articles.csv";
+
+    public ArrayList<Article> readArticlesFromFile() throws IOException, ClassNotFoundException {
```

Comment:
Powinineś zwracać List<Aritcle>, jeżeli jakaś klasa ma interfejs to ten interfejs służy do komunikacji z tą klasą bo po to jest interfejs -> to jest taki uniwersalny pilot do telewizora, który obsługuje różne telewizory. Czyli pilotem jest List a telewizorem ArrayList albo LinkedList